tests/e2e/export.go

```diff
@@ -15,6 +15,7 @@
 package e2e
 
 import (
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Could you run `goimports -w tests/` to fix the import grouping?

I also created #1497 .